src/go/rdctl/cmd/snapshot.go

```diff
@@ -70,10 +69,6 @@ func wrapSnapshotOperation(cmd *cobra.Command, appPaths paths.Paths, resetOnFail
 		return err
 	}
 	if err := wrappedFunction(); err != nil {
-		if resetOnFailure {
-			factoryreset.DeleteData(appPaths, true)
-			return err
-		}
```

Comment:
Removing these lines will have the effect of trying to restart the backend after `manager.Restore()` fails and the files are deleted. This isn't what we want, is it?

Reply:
No, it is not.